eox_nelp/api_clients/pearson_rti.py

```diff
@@ -122,41 +109,36 @@ def import_exam_authorization(self, payload):
         Returns:
             <Dict>: Parsed xml response to Dict format.
         """
-        @audit_method(action="Import Exam Authorization")
-        def import_exam_authorization_request(payload):
-            """This is a wrapper that allows to make audit-able the import_exam_authorization method."""
-            xml_response = BeautifulSoup(
-                self.make_post("cxfws2/services/EADService", payload),
-                "xml",
-            )
-            fault = xml_response.find("soapenv:Fault")
-
-            # There are multiple kind of errors, some of them generates a soapenv:Fault section and others
-            # generates an status error section, the following condition handles the first case.
-            if fault:
-                return {
-                    "status": "error",
-                    "fault_code": xml_response.find("faultcode").text,
-                    "message": xml_response.find("faultstring").text,
-                }
-
-            status = xml_response.find("status")
-
-            if status:
-                return {
-                    "status": status.text.lower(),
-                    "message": xml_response.find("message").text if xml_response.find("message") else "",
-                    "client_candidate_id": xml_response.find("clientCandidateID").text,
-                }
-
-            LOGGER.error(
-                "An unexpected error has occurred trying to make a EAD request getting the following response: %s",
-                xml_response,
-            )
+        xml_response = BeautifulSoup(
+            self.make_post("cxfws2/services/EADService", payload),
+            "xml",
+        )
+        fault = xml_response.find("soapenv:Fault")
 
+        # There are multiple kind of errors, some of them generates a soapenv:Fault section and others
+        # generates an status error section, the following condition handles the first case.
+        if fault:
             return {
```

Comment:
Maybe here we could add an exception. To try to add the eox_audit with the failure status instead of success.
 I was thinking of the audit error raising a pearsonExceptionm but it would be twice I think so... or may be not.
We have to view it after the PearsonException PR works...